src/Services/InspectionsApi.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Dxw\Whippet\Services;
+
+// Responsible for calling an API and returning information about dxw inspections
+// associated with a plugin
+class InspectionsApi
+{
+    public function __construct(
+        $host,
+        \Dxw\Whippet\Services\JsonApi $json_api
+    ) {
+        $this->host = $host;
+        $this->jsonApi = $json_api;
+    }
+
+    public function get_inspections($plugin_slug)
+    {
+        $result = $this->jsonApi->get($this->url($plugin_slug));
+        if ($result->isErr()) {
+            return $result;
+        }
+        $inspections = $this->buildInspections($result->unwrap());
+        return \Result\Result::ok($inspections);
+    }
+
+    private function buildInspections($raw_inspections)
+    {
+        return array_map(function ($raw_inspection) {
+            return new Inspection(
+                $raw_inspection['date'],
+                $raw_inspection['versions'],
+                $raw_inspection['result'],
+                $raw_inspection['url']
+            );
+        }, $raw_inspections);
+    }
+
+    private function url($plugin_slug)
+    {
+        return $this->host.'/wp-json/v1/inspections/'.$plugin_slug;
```

Comment:
I went a fair way down this path, but it seems like whippet.json is strongly associated with dependencies: the hash of that file is used to determine whether the lockfile needs to be updated, so changing the API host or Path results in the whole lock file needing to be updated, which feels wrong to me.

What I _have_ done is move the _path_ definition to the same place as the host. 

Although it would be best to manage these URLs as config, I feel like they are going to be fairly stable, so could we consider shipping this as-is?